ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -337,6 +343,15 @@ public CompletableFuture<PutResult> put(NamedBlobRecord record, NamedBlobState s
         }, null);
   }
 
+  @Override
+  public CompletableFuture<PutResult> updateBlobStateToReady(NamedBlobRecord record) {
+    return executeTransactionAsync(record.getAccountName(), record.getContainerName(), true,
```

Comment:
we should have a metric for this operation just like any other operations.

Reply:
Added.